omp/reorder/rcm_kernels.cpp

```diff
@@ -235,8 +235,8 @@ struct UbfsLinearQueue {
 #define GKO_CMPXCHG_IMPL(ptr, ptr_expected, replace_with) \
     return __atomic_compare_exchange_n(                   \
```

Comment:
clang and gcc indeed use int for memory_order although clang doc uses `__atomic_compare_exchange_N` not `__atomic_compare_exchange_n`

Reply:
Should I fix that line as well? Or is it something for other authors/maintainers?

Reply:
It should be okay now